docs/glossary.md

```diff
@@ -0,0 +1,168 @@
+ Glossary of Technical Terms
+---
+
+## A
+- [Account Aliases](#account-aliases)
+- [Admin Transaction](#admin-transaction)
+- [Auto Associations](#asynchronous)
+- [Auto-Renew Accounts](#auto-renew-accounts)
+- [Address Book](#address-book)
+- [Application Accounts](#application-accounts)
+
+## B
+- [BLS](#bls)
+- [Block Proof](#block-proof)
+- [Block Signature](#block-signature)
+- [Block Stream](#block-stream)
+
+## C
+- [Civilian Accounts](#civilian-accounts)
+- [Config.txt](#config.txt)
+- [Congestion Pricing](#congestion-pricing)
+- [Consensus Time](#consensus-time)
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transaction](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The _**network**_ address book is data structure that contains information (such as node ID, alias, IP address, and weight) for all nodes in the network, 
+not just consensus, but also block nodes, future validator nodes and more.
+The address book is stored as a file on disk at the moment but will be stored purely in state as part of Dynamic Address Book phase 3.	Disambiguation: There is also an "address book" in files 0.0.101 and 0.0.102 which is different to the "network address book" which are manually managed by devops.
```

Comment:
I am not sure the community understands what "dynamic address book phase 3" means. I don't think we have a HIP yet for that. I guess maybe when it becomes available we could link it for reference. Maybe we should define what a "Dynamic Address Book" is and link to that. 

Suggestion:

**Dynamic Address Book**: The envisioned state where the system can create, update, or delete nodes in the address book through Hedera transactions. In this future state, rather than manually updating nodes during predefined upgrade periods, the address book can be managed in real-time with the following transaction types:

[NodeCreate](https://github.com/hashgraph/hedera-protobufs/blob/main/services/node_create.proto): Adds a new node to the address book.
[NodeUpdate](https://github.com/hashgraph/hedera-protobufs/blob/main/services/node_update.proto): Modifies the details of an existing node.
[NodeDelete](https://github.com/hashgraph/hedera-protobufs/blob/main/services/node_delete.proto): Removes a node from the address book.

The actions of the transactions will still take effect at the node upgrade window [if I understood correctly].

Reply:
> These might work better as definition lists, by the way. Here's an example (the spacing was off also, so it's a good example).

Thanks @jsync-swirlds. I will implement this formatting in a separate PR 👍 

> I am not sure the community understands what "dynamic address book phase 3" means. I don't think we have a HIP yet for that. I guess maybe when it becomes available we could link it for reference. Maybe we should define what a "Dynamic Address Book" is and link to that.

Good suggestion, have added. Thanks @SimiHunjan.

Reply:
Will address styling, formatting and definition lists in a future PR